introduction_to_amazon_algorithms/jumpstart-foundation-models/text2text-fine-tuning-flan-t5.ipynb

```diff
@@ -0,0 +1,742 @@
+{
```

Comment:
<p>I guess the entire dataset is in this json file? I think it is best to put it in our specialized notebook bucket, especially if rajpurkar is an external party or leaves the company in the future. Should be quick to do</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3916/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Actually, is it possible to use s3://sagemaker-sample-files/datasets/text/squad/train-v1.1.json instead?</p>

Reply:
we need v2 of this dataset (squad2.0)